spec/linter-stylelint-spec.js

```diff
@@ -7,117 +7,136 @@ describe('The stylelint provider for Linter', () => {
 
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
+    atom.config.set('linter-stylelint.usePreset', true);
+
     waitsForPromise(() => {
-      atom.packages.activatePackage('linter-stylelint');
```

Comment:
Looks like the linter-php specs are broken in v1.4.0-beta3 even as they are, so _yay_. Seeing if I can get those working since it would be nice if we can get an activation method that works for packages both with and without `activationHooks`.